src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -72,14 +72,13 @@ for (const section of sidebarSections) {
 <style>
 	nav {
 		width: 100%;
-		margin-inline-end: 1rem;
 		padding-top: 0rem;
 	}
 	.nav-groups {
 		height: 100%;
 		overflow-x: visible;
 		overflow-y: auto;
-		max-height: calc(100vh - var(--theme-navbar-height));
+		max-height: calc(var(--cur-viewport-height) - var(--theme-navbar-height));
```

Comment:
I’m seeing one tricky bug that I think _might_ come back to this. Basically on mobile where we have the issue that `100vh` can be misleading, I’m sometimes seeing the header pull down _below_ the top of the screen when the menu is open. Can’t reproduce in the iOS simulator and can’t figure out how to capture my phone’s screen, but the flow is:

1. Scroll down the page a little so the address bar vanishes
2. Open the hamburger menu
3. Scroll up the nav menu

Result (note the dark empty space at the top):

<img src="https://user-images.githubusercontent.com/357379/177363172-0a375327-b07c-43a6-8dba-4d02bea10a00.jpg" width="300">

Browser: Firefox
OS: Android 11

Can’t reproduce in Chrome on my phone so may be FF only? (This could also be a genuine bug with mobile FF when the address bar is at the bottom of the screen.)

Reply:
Hmm, can you reproduce this issue on the old site with this browser as well?

Whenever I noticed some hiccups of the left sidebar (and the hamburger menu is just a differently styled left sidebar), I checked the old page and saw that it also had the same issues. Unfortunately, the "incorrect height of sidebar" issues are not easy to fix, and if I wasn't able to, I let them be because at least I didn't make things worse. 😅

Reply:
Not present on the old site. With the current production version, the address bar never hides so I guess there’s never this bug that `100vh` mismatches with the actual viewport height.

You don’t see this is iOS Safari, I guess?

Reply:
No, it works fine there! I don't have any device to reproduce this with. 😬

Reply:
Haha! Now you know how it feels 🤪 

Let me try using `100dvh` on supported browsers locally and see if that fixes it (https://caniuse.com/viewport-unit-variants).

Reply:
My var already uses dvh if supported 😅

Reply:
Doh… just wrote out an identical `@supports` block to what you had 😆 

Maybe we want `lvh` or `svh` then? 🤔

Reply:
Nope same behaviour with both.

Reply:
OK. So not sure what the problem is here. Let’s leave it as BFMPTWWHB (Better For Most People Than What We Had Before). If we figure out a fix in the future, that’s great, but FF Mobile has a market share of something like 0.4% and it’s a harmless rendering glitch so should be OK.